example/service/localstack_endpoints.json

```diff
@@ -1,17 +1,17 @@
 {
     "APIGateway": "http://localhost:4567",
-    "CloudFormation": "http://localstack:4581",
```

Comment:
See the comment above regarding the use of compose.

I'd prefer to stick with the localstack endpoint references (maybe it requires 2 config files/examples?)